.env.example

```diff
@@ -1,5 +1,6 @@
 # Repo api:
-API_URL=
-AUTH_TOKEN=
+API_URL= https://api.repo.nypl.org
+DC_URL=https://qa-digitalcollections.nypl.org
+AUTH_TOKEN= # Get credentials from a teammate!
```

Comment:
I think you need to define these in the Dockerfile: https://github.com/NYPL/digital-collections/blob/main/Dockerfile#L18C4-L18C4

I think task definition passes the env var down when docker is building the app but Docker needs to read and pass it down to the app. You can try without it first and then with it.